app/public/index.html

```diff
@@ -8,6 +8,7 @@
 
     <link rel="icon" type="image/png" href="favicon.png" />
     <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="global-colors.css" />
```

Comment:
Reads just as well with just `colors.css`.

Reply:
You're right 👍